user-interface/src/lib/components/combobox/ComboBox.tsx

```diff
@@ -59,18 +60,27 @@ function ComboBoxComponent(props: ComboBoxProps, ref: React.Ref<ComboBoxRef>) {
     wrapPills,
     ariaLabelPrefix,
     ariaDescription,
+    value: _value,
```

Comment:
Why are we doing this and why are we using props.value elsewhere in this component if _value is equivalent?